### ISSUE TITLE
List of instance ids, from vswitch data block, consist additional "" values.

### ISSUE THREAD

**Issue by GabrielZabielski** [terraform-providers/terraform-provider-alicloud#1582]
Hi there,

I've got a simple bug for you, have fun.

### Terraform Version
Terraform v0.12.6
+ provider.alicloud v1.55.0

### Affected Resource(s)
- data.alicloud_vswitches

### Terraform Configuration Files
```hcl
data "alicloud_vswitches" "vswitch" {}
output "vsw" {
  value = data.alicloud_vswitches.vswitch
}
```

### Expected Behavior
```hcl
vsw = [
    {
      "cidr_block" = "10.0.0.0/24"
      "creation_time" = "2019-08-12T13:05:55Z"
      "description" = "vswitch"
      "id" = "vsw-gwTTTTTTTTTTTTTTTTTTT"
      "instance_ids" = [
        "i-gwZZZZZZZZZZZZZZZZZZ",
        "i-gwSSSSSSSSSSSSSSSSSS",
        "i-gwEEEEEEEEEEEEEEEEEE",
        "i-gwRRRRRRRRRRRRRRRRRR",
      ]
      "is_default" = false
      "name" = "vswitch_1"
      "vpc_id" = "vpc-gwWWWWWWWWWWWWWWWWWWW"
      "zone_id" = "eu-central-1a"
    },
    {
      "cidr_block" = "10.0.1.0/24"
      "creation_time" = "2019-08-08T14:30:03Z"
      "description" = "vSwitch "
      "id" = "vsw-gwGGGGGGGGGGGGGGGGGGG"
      "instance_ids" = [
        "i-gwYYYYYYYYYYYYYYYYYY",
        "i-gwXXXXXXXXXXXXXXXXXX",
      ]
      "is_default" = false
      "name" = "vswtic2"
      "vpc_id" = "..."
      "zone_id" = "..."
    },
 ...
]
```

### Actual Behavior
```hcl
vsw = [
    {
      "cidr_block" = "10.0.0.0/24"
      "creation_time" = "2019-08-12T13:05:55Z"
      "description" = "vswitch"
      "id" = "vsw-gwTTTTTTTTTTTTTTTTTTT"
      "instance_ids" = [
        "",
        "", 
        "",
        "",
        "i-gwZZZZZZZZZZZZZZZZZZ",
        "i-gwSSSSSSSSSSSSSSSSSS",
        "i-gwEEEEEEEEEEEEEEEEEE",
        "i-gwRRRRRRRRRRRRRRRRRR",
      ]
      "is_default" = false
      "name" = "vswitch_1"
      "vpc_id" = "vpc-gwWWWWWWWWWWWWWWWWWWW"
      "zone_id" = "eu-central-1a"
    },
    {
      "cidr_block" = "10.0.1.0/24"
      "creation_time" = "2019-08-08T14:30:03Z"
      "description" = "vSwitch "
      "id" = "vsw-gwGGGGGGGGGGGGGGGGGGG"
      "instance_ids" = [
        "",
        "",
        "i-gwYYYYYYYYYYYYYYYYYY",
        "i-gwXXXXXXXXXXXXXXXXXX",
      ]
      "is_default" = false
      "name" = "vswtic2"
      "vpc_id" = "..."
      "zone_id" = "..."
    },
   ...
]
```

### Steps to Reproduce
1. Any VPC with vswitch and an ECS instance in,
2. `terraform apply` on the `Terraform Configuration Files`

**Comment by Lexsss** [comment]
Hi @GabrielZabielski , thx for supporting our project, this issue has been resolved in v1.55.1, please check it.

**Comment by GabrielZabielski** [comment]
Yes, it's. Thanks guys!
